cds_dojson/marc21/fields/books/book.py

```diff
@@ -31,8 +35,62 @@
 @filter_values
 def alternative_titles(self, key, value):
     """Alternative titles."""
+    if 'n' in value:
+        self['volume'] = volume(self, key, value)
```

Comment:
curiosity, why not:
```suggestion
        self['volume'] = self.volume(self, key, value)
```

Reply:
it could be more like `self.rules['volume'](...` (not sure of exact form) but it gets a bit unreadable for me then